docs/manual/lifecycle/life.ipynb

```diff
@@ -885,7 +885,7 @@
     "it is safe to ignore for general application-level code.\n",
     "\n",
     "For more information, see the [`@register_passable`\n",
-    "documentation](/mojo/manual/decorators/register-passable.html).\n",
+    "documentation](/mojo/manual/decorators/register_passable.html).\n",
```

Comment:
No, this is correct as it is.

Reply:
Sorry, I closed this PR and re-create new one here https://github.com/modularml/mojo/pull/1474

Can you help check?